staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -128,20 +131,24 @@ func (s *Serializer) Decode(originalData []byte, gvk *schema.GroupVersionKind, i
 		return nil, nil, err
 	}
 
-	if gvk != nil {
-		// apply kind and version defaulting from provided default
+	// apply kind and version defaulting from provided default
+	gvkDefault := func(actual *schema.GroupVersionKind, defaultGVK schema.GroupVersionKind) {
```

Comment:
how about `completeGVK`?

Reply:
in other words: gvkDefault should be an imperative with an obvious verb.

Reply:
done